ingest-app/src/cmr/ingest/api/translation.clj

```diff
@@ -32,14 +36,48 @@
                                  :umm-json
                                  (umm-spec/generate-metadata context umm :umm-json))))
 
+(def xsl-transformer-cache-name
+  "This is the name of the cache to use for XSLT transformer templates. Templates are thread
+  safe but transformer instances are not.
+  http://www.onjava.com/pub/a/onjava/excerpt/java_xslt_ch5/?page=9"
```

Comment:
this URL redirects me to https://www.oreilly.com/ideas?page=9 which doesn't have any documentation.

Reply:
It is existing code that was copied from search-app. The link must have become obsolete since. I will remove it.